lib/elixir/lib/access.ex

```diff
@@ -422,4 +422,58 @@ defmodule Access do
   defp all([], _next, gets, updates) do
     {:lists.reverse(gets), :lists.reverse(updates)}
   end
+
+  @doc ~S"""
+  Accesses the _nth_ element (zero based) of a list.
+
+  ## Examples
```

Comment:
We need to document and add examples when the index is out of bounds in both get and update. It seems in update the function is never called... which should be documented.